elide-contrib/elide-dynamic-config-helpers/src/main/resources/elideTableSchema.json

```diff
@@ -0,0 +1,296 @@
+        },
+        "readAccess": {
+          "title": "Dimension read access",
+          "description": "Read permission for the metric.",
+          "type": "string",
+          "default": "Allow All"
+        },
+        "definition": {
+          "title": "Dimension definition",
+          "description": "The definition of the dimension",
+          "type": "string"
+        },
+        "type": {
+          "title": "Dimension field type",
+          "description": "The data type of the dimension field",
+          "type": "string",
+          "enum": [
+            "time",
+            "text",
+            "coordinate",
+            "number",
+            "boolean"
+          ],
+          "default": "text"
+        },
+        "tags": {
+          "title": "Dimension tags",
+          "description": "An array of string based tags for dimensions",
+          "type": "array",
+          "uniqueItems": true,
+          "items": {
+            "type": "string"
+          },
+          "default": []
+        },
+        "grains": {
+          "title": "Dimension grains",
+          "description": "Time definition granularity",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      },
+      "required": [
+        "name",
+        "definition"
+      ]
+    }
+  },
+  "type": "object",
+  "properties": {
+    "tables": {
+      "title": "Elide Table Models",
+      "description": "Array of elide Table Models",
+      "type": "array",
+      "uniqueItems": true,
+      "items": {
+        "properties": {
+          "name": {
+            "title": "Table Model Name",
+            "description": "The name of the model. This will be the same as the POJO class name.",
+            "type": "string",
+            "pattern": "^[A-Z][A-Za-z]*$"
+          },
+          "schema": {
+            "title": "Table Schema",
+            "description": "The database or schema where the model lives.",
+            "type": "string",
+            "pattern": "^([A-Za-z]*_?[A-Za-z]*)*$",
+            "default": ""
+          },
+          "table": {
+            "title": "Table name",
+            "description": "The physical table name where the model lives.",
+            "type": "string",
+            "pattern": "^([A-Za-z]*_?[A-Za-z]*)*$",
+            "default": ""
+          },
+          "hidden": {
+            "title": "Hide/Show Table",
+            "description": "Whether this table is exposed via API metadata",
+            "type": "boolean",
+            "default": false
+          },
+          "sql": {
+            "title": "Table SQL",
+            "description": "SQL query which is used to populate the table.",
+            "type": "string",
+            "default": ""
+          },
+          "description": {
+            "title": "Table Model description",
+            "description": "A long description of the model.",
+            "type": "string",
+            "default": "[$name]"
```

Comment:
I haven't seen this before.  Does this actually reference the name?  Can you point me to the docs for this? Thanks.

Reply:
I read about it in a custom Json implementation and tried to use it. However turns out there is no way to do that as of now. So it is going to be handled in the code.